client/gcp/gcp.go

```diff
@@ -0,0 +1,218 @@
+package gcp
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"regexp"
+	"strconv"
+	"strings"
+
+	"github.com/bytebase/dbcost/client"
+)
+
+// Client is the client struct
+type Client struct{}
+
+// NewClient return a client
+func NewClient() Client {
+	return Client{}
+}
+
+const rdsServiceID = "9662-B51E-5089"
+
+// this apiKey is for demo purpose only
+const apiKey = "AIzaSyAj36rmGvQ_3kBqd6Lp3nSzPDo2_nD-sdM"
+
+// priceInfoEndpoint is the endpoint for all Cloud SQL services on GCP
+// For more information, please refer to https://cloud.google.com/billing/v1/how-tos/catalog-api
+var priceInfoEndpoint = fmt.Sprintf("https://cloudbilling.googleapis.com/v1/services/%s/skus?key=%s", rdsServiceID, apiKey)
+
+type unitPrice struct {
+	CurrencyCode string  `json:"currencyCode"`
+	Unit         string  `json:"units"`
```

Comment:
Per previous comment, add explaination or examples about what Unit and Nano's are.